your-first-cluster.md

```diff
@@ -101,9 +106,9 @@ spec:
     serviceDomain: "cluster.local"
   topology:
     class: quick-start                   # The name of the ClusterClass the Cluster is templated from.
+    version: v1.24.6                     # The version of the Kubernetes Cluster.
```

Comment:
Just moved it up as it seemed more "high-level" / important than the things below